src/store/modules/ProfileModule.ts

```diff
@@ -292,6 +302,11 @@ export default {
             commit('setDisabledPosition', settings.getInstalledDisablePosition());
         },
 
+        async reset({commit, rootGetters}) {
+            await rootGetters['settings'].setProfile('Default');
```

Comment:
We can't access settings here if this is called every time in the create hook of GameSelectionScreen, as this will cause an error when the app is first opened, causing the settings to not be loaded (e.g. game list always rendered as list instead of cards).

There are multiple ways how to approach fixing this but I'd rather not dive into figuring out the best (or even a working) way to solve the issue, so give it a try and let's check it together if you come up empty.

Reply:
Now the behaviour is "fixed", but IDK what'd be the best for the UX; resetting the profile everytime or not.

Reply:
From the top of my head I'd say we don't need to reset the profile on profile selection screen unless the selected profile changes (which I guess means we override all the relevant values anyway, so no reset is needed in that case either). On the game selection screen always resetting the profile is ok methinks.

Reply:
And for the record, the current solution no longer causes the error so in that sense this issue is resolved.